cobigen-cli/src/test/java/com/devonfw/cobigen/cli/commandtests/GenerateCommandTest.java

```diff
@@ -104,25 +98,24 @@ public void generateFromOpenApiTest() throws IOException {
         args[4] = "--increments";
         args[5] = "1,15,22";
 
-        // Act
         CobiGenCLI.main(args);
-
-     // Assert
-     		Path rootPath = outputRootFile.toPath();
-     		File generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/salemanagement").toFile();
-     		geneatedList.add(generatedFiles);
-     		generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/shopmanagement").toFile();
-     		geneatedList.add(generatedFiles);
-     		generatedFiles = new File(testFileRootPath + "/devon4ng-ionic-application-template");
-     		geneatedList.add(generatedFiles);
-     		geneatedList.add(rootPath.resolve("src").toFile());
-     		geneatedList.add(rootPath.resolve("docs").toFile());
-     		geneatedList.add(outputRootFile.getParentFile().toPath().resolve("api").toFile());
-     		GenerateCommandTest.deleteGeneratedFiles(geneatedList);
-     		geneatedList.clear();
+        
+ 		Path rootPath = outputRootFile.toPath();
+ 		File generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/salemanagement").toFile();
+ 		geneatedList.add(generatedFiles);
+ 		generatedFiles = rootPath.resolve("src/main/java/com/devonfw/angular/test/shopmanagement").toFile();
+ 		geneatedList.add(generatedFiles);
+ 		generatedFiles = new File(testFileRootPath + "/devon4ng-ionic-application-template");
+ 		geneatedList.add(generatedFiles);
+ 		geneatedList.add(rootPath.resolve("src").toFile());
+ 		geneatedList.add(rootPath.resolve("docs").toFile());
+ 		geneatedList.add(outputRootFile.getParentFile().toPath().resolve("api").toFile());
+ 		GenerateCommandTest.deleteGeneratedFiles(geneatedList);
+ 		geneatedList.clear();
```

Comment:
The indentation in all of these lines is wrong, it should be on the same level as `CobiGenCLI.main(args);`